CHANGELOG.md

```diff
@@ -1,5 +1,38 @@
 # Quarkus Tools for Visual Studio Code Changelog
 
+## [1.19.0](https://github.com/redhat-developer/vscode-quarkus/milestone/22?closed=1) (October 22, 2024)
+
+### Enhancements
+
+- `quarkus.rest.path` support. See [quarkus-ls#1003](https://github.com/redhat-developer/quarkus-ls/pull/1003).
+- Completion on available Ollama models support for `quarkus.langchain4j.ollama.chat-model.model-id` property value. See [quarkus-ls#985](https://github.com/redhat-developer/quarkus-ls/pull/985).
+
+### Bug Fixes
+
+- Fix Message: Request qute/java/documentLink failed with message: Cannot invoke "org.eclipse.jdt.core.dom.ITypeBinding.getQualifiedName()" . See [#839](https://github.com/redhat-developer/vscode-quarkus/issues/839).
+- Fix false-positive error when string parameter contains a comma. See [quarkus-ls#1011](https://github.com/redhat-developer/quarkus-ls/issues/1011).
+- Support for Vert.x Integration. See [quarkus-ls#1009](https://github.com/redhat-developer/quarkus-ls/issues/1009).
+- TemplateExtension match any support. See [quarkus-ls#1008](https://github.com/redhat-developer/quarkus-ls/pull/1008).
+- Fix false positive error with param name and user tag. See [quarkus-ls#1007](https://github.com/redhat-developer/quarkus-ls/pull/1007).
+- Fix add languageId filter for options similar to microprofile. See [quarkus-ls#999](https://github.com/redhat-developer/quarkus-ls/pull/999).
+- Fix Message: Request qute/java/documentLink failed with message: Cannot invoke "org.eclipse.jdt.core.dom.ITypeBinding.getQualifiedName()". See [quarkus-ls#996](https://github.com/redhat-developer/quarkus-ls/pull/996).
+- Fix NPE with {# and Qute inlay hint. See [quarkus-ls#991](https://github.com/redhat-developer/quarkus-ls/issues/991).
+- Cannot publish to OpenVSX due to out of date NodeJS. See [vscode-microprofile#303](https://github.com/redhat-developer/vscode-microprofile/issues/303).
```

Comment:
This can probably go under build, and I would list the issue for this repo: https://github.com/redhat-developer/vscode-quarkus/issues/857

Reply:
fixed